Gemfile

```diff
@@ -1,32 +1,35 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.2.2"
+ruby '3.2.2'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
```

Comment:
- [ ] I was unable to fetch any greeting from the front end due to a CORS policy issue. You need to install `rack-cors`, and create a `config/initializers/cors.rb` file, and configure the allowed requests with the snippet below. You can see [the rack-cors documentation](https://github.com/cyu/rack-cors).

  ```
  Rails.application.config.middleware.insert_before 0, Rack::Cors do
    allow do
      origins '*'
      resource '*', headers: :any, methods: [:get, :post, :patch, :put]
    end
  end
  ```

  ![deoe](https://github.com/aubinleyoung/hello-rails-back-end/assets/36542986/a2c65f3b-be83-492f-81d5-bc463a125dec)